adam-pipeline/wrapper.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import os
+import shutil
+import subprocess
+import json
+import logging
+from uuid import uuid4
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger()
+
+
+def write_config(mount, args):
+    
+    # open path to write config file at
+    path = "adam-preprocessing-%s.config" % (str(uuid4()))
+    fp = open(path, 'w')
+
+    # generate config
+    config = '''# ADAM preprocessing pipeline configuration file; autogenerated, do not modify!
+num-nodes:
+master-ip:
+dbsnp: %s
+memory: %s
+run-local: true
+local-dir: %s
+''' % (args.known_sites, args.memory, mount)
+
+    # write config to file
+    print >> fp, config
+
+    # flush and close file
+    fp.flush()
+    fp.close()
+
+    # return config file name
+    return path
+
+
+def call_pipeline(mount, args):
+    
+    # get uuid and make a work directory
+    uuid = 'Toil-Adam-' + str(uuid4())
```

Comment:
I'd prefer `toil-adam-`. I hardly ever use upper-case in snake-case identifiers.